src/server/status/status.js

```diff
@@ -55,9 +55,11 @@ export default class Status extends EventEmitter {
   toJSON() {
     return {
       id: this.id,
-      state: this.state,
-      icon: states.get(this.state).icon,
-      message: this.message,
+      state: {
```

Comment:
/cc @elastic/kibana-monitoring any idea if this is going to impact the beats module or status poller?

Reply:
The only usage I've found besides the status page / checking logic itself is the api_debug CLI dev tool in xpack. It doesn't do anything with the data back from the API besides dump it into the console and I've confirmed it still works.

Dropped in Monitoring's slack to confirm, but I think we will be good to go.